java-tools/OsmAndServer/src/main/java/net/osmand/server/controllers/pub/GeotiffTileController.java

```diff
@@ -35,15 +35,15 @@ public class GeotiffTileController {
 	private static final String SLOPE_TYPE = "slope";
 	private static final String HEIGHT_TYPE = "height";
 
+	private static final long SEVEN_DAYS_IN_MILLIS = TimeUnit.SECONDS.toMillis(7);
+	private static final long CLEANUP_INTERVAL_MILLIS = 12 * 60 * 60 * 1000L; // 12 hours
```

Comment:
Does it really need java.util.concurrent.TimeUnit to calculate 7 days in milliseconds?